File: 3685-count-subarrays-of-length-three-with-a-condition/3685-count-subarrays-of-length-three-with-a-condition.cs
public class Solution {
    public int CountSubarrays(int[] nums) {
        var n = nums.Length;
        var cnt = 0;
        for (int i = 1; i < n  - 1; ++i) {
            if (2 * (nums[i - 1] + nums[i + 1]) == nums[i])
                cnt++;
        }
        return cnt;
    }
}

File: 3685-count-subarrays-of-length-three-with-a-condition/3685-count-subarrays-of-length-three-with-a-condition.java
class Solution {
    public int countSubarrays(int[] nums) {
        int n = nums.length;
        int cnt = 0;
        for (int i = 1; i < n - 1; ++i) {
            if (2 * (nums[i - 1] + nums[i + 1]) == nums[i])
                cnt++;
        }
        return cnt;
    }
}